src/components/Identity/index.js

```diff
@@ -35,6 +36,9 @@ const createIdentity = ({ config, logger, cookieJar }) => {
   let lifecycle;
   const idSyncs = createIdSyncs(config, logger, cookieJar);
   let alreadyQueriedForIdSyncs = false;
+  // Defining initial state of customerIds
+  let customerIdState = {};
```

Comment:
Do we need both? isn't a non-empty state an indication that we have customer IDs?

Reply:
Yes, i was trying to avoid the expensive operation to determine if the object is empty - and it would happen on every request.